doc/source/ray-contribute/docs.ipynb

```diff
@@ -64,6 +65,7 @@
     "\n",
     "Don't use `-U` in this step. You don't want to upgrade dependencies because `requirements-doc.txt` pins exact versions you need to build the docs.\n",
     "\n",
+    "### Run make\n",
```

Comment:
I think this should capture the logical definition of the step, rather than the CLI command name. 

```suggestion
    "### Build documentation\n",
```

Reply:
"Run make" is like replacing "Install dependencies" with "Run pip install"